### PR TITLE
33 update side bar app bar footer

### PR DESCRIPTION
**Pull request by megbouv**

### CODE REVIEW

File: src/common/Bottom.vue [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-  export default {
-    name: 'Bottom',
-    data: () => ({
-      icons: ['fab fa-facebook', 'fab fa-twitter', 'fab fa-google-plus', 'fab fa-linkedin', 'fab fa-instagram']
-    }),
-    props: []
-  }
+export default {
+  name: "Bottom",
+  data: () => ({
```
Comment (mattstryfe) [inline src/common/Bottom.vue:27]: Can you change this to the proper syntax while you're in here?

File: src/App.vue [hunk 2]
```diff
@@ -1,99 +1,38 @@
...
+  components: { AppBar, SideBar, Bottom },
   props: {
     source: String
   },
-  data () {
+  data() {
     return {
-      drawer: false,
+      drawer: true,
       dialog: false
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/App.vue:30]: yes, removed

File: src/App.vue [hunk 3]
```diff
@@ -1,99 +1,38 @@
...
     </v-content>
 
     <!-- Bootom: located in common/ -->
-    <Bottom/>
-
+    <Bottom />
   </v-app>
 </template>
 <script>
-import Bottom from '@/common/Bottom'
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/App.vue:54]: Renamed

File: src/common/AppBar.vue [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  border-radius: 0px !important;
+}
+#app {
+  font-family: "Candara", Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+}
+#nav {
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/common/AppBar.vue:76]: done

File: src/common/AppBar.vue [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  padding: 30px;
+}
+
+#nav a {
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/common/AppBar.vue:80]: done

File: src/common/AppBar.vue [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+  font-weight: bold;
+  color: #2c3e50;
+}
+
+#nav a.router-link-exact-active {
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/common/AppBar.vue:85]: done

File: src/common/AppBar.vue [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    }
+  }
+};
+</script>
+
+<style scoped>
+.no-underline {
+  text-decoration: none;
+}
+.custom-no-radius {
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/common/AppBar.vue:66]: This needs to stay or else the Project Bluefire in AppBar gets underlined

File: src/common/AppBar.vue [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+    <v-btn icon disabled>
+      <v-icon>fa-user</v-icon>
+    </v-btn>
+  </v-app-bar>
+</template>
+
+<script>
+export default {
+  name: "AppBar",
+
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/common/AppBar.vue:43]: done

File: src/common/AppBar.vue [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+</script>
+
+<style scoped>
+.no-underline {
+  text-decoration: none;
+}
+.custom-no-radius {
+  border-radius: 0px !important;
+}
+#app {
```
[the remark this answers is not in the document]
Comment (megbouv) [reply src/common/AppBar.vue:69]: done

File: src/common/Bottom.vue (hunk 1 above)
Comment (megbouv) [reply src/common/Bottom.vue:27]: Did you mean change name to BottomBar?

Comment (mattstryfe) [reply src/common/Bottom.vue]: That too.  But it should be 
`data() {
  return { }
}
`

**Pull request merged**